dsa-23au/java-dsa/heaps-trees/src/main/BinaryTree.java

```diff
@@ -0,0 +1,34 @@
+package dev.codewithfriends;
+
+private static class binaryNode<T> 
+{
+    private T data;
```

Comment:
This code helps me understand the concepts of graphs and Binary search trees especially at a pseudocode/design perspective.